From 635fa66abe6e502c9b78b1dc66757bf67fd163e1 Mon Sep 17 00:00:00 2001
From: Imre Deak <imre.deak@nokia.com>
Date: Wed, 22 Apr 2009 14:40:48 +0200
Subject: [PATCH 52/69] DSS2: VRAM: fix section mismatch warning

postponed_regions are accessed from the non __init
omap_vram_add_region().

Signed-off-by: Imre Deak <imre.deak@nokia.com>
---
 arch/arm/plat-omap/vram.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-omap/vram.c b/arch/arm/plat-omap/vram.c
index e847579..b126a64 100644
--- a/arch/arm/plat-omap/vram.c
+++ b/arch/arm/plat-omap/vram.c
@@ -63,11 +63,11 @@
 #define MAX_POSTPONED_REGIONS 10
 
 static bool vram_initialized;
-static int postponed_cnt __initdata;
+static int postponed_cnt;
 static struct {
 	unsigned long paddr;
 	size_t size;
-} postponed_regions[MAX_POSTPONED_REGIONS] __initdata;
+} postponed_regions[MAX_POSTPONED_REGIONS];
 
 struct vram_alloc {
 	struct list_head list;
-- 
1.6.2.4

